ros2-library/src/test/java/us/ihmc/ros2/example/NonRealtimeROS2TalkerExample.java

```diff
@@ -1,43 +1,46 @@
 /*
  * Copyright 2017 Florida Institute for Human and Machine Cognition (IHMC)
- * 
+ *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
  *     http://www.apache.org/licenses/LICENSE-2.0
- *     
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
- * limitations under the License. 
+ * limitations under the License.
  */
 package us.ihmc.ros2.example;
 
 import std_msgs.msg.dds.String;
-import us.ihmc.pubsub.DomainFactory;
 import us.ihmc.ros2.ROS2Node;
+import us.ihmc.ros2.ROS2NodeBuilder;
+import us.ihmc.ros2.ROS2NodeBuilder.SpecialTransportMode;
 import us.ihmc.ros2.ROS2Publisher;
 
 import java.io.IOException;
 
 /**
  * Java version of the ROS2 demo listener.
- * 
- * To test, start a ROS2 talker using 
- * 
+ *
+ * To test, start a ROS2 talker using
+ *
  *    ros2 run demo_nodes_cpp talker -- -t chatter
- * 
+ *
  * @author Jesper Smith
  *
  */
 public class NonRealtimeROS2TalkerExample
 {
    public static void main(String[] args) throws IOException, InterruptedException
```

Comment:
Just to fix this broken example...
```suggestion
   public static void main(java.lang.String[] args) throws IOException, InterruptedException
```

Reply:
Nice catch, fixed